nexus/db-queries/src/db/datastore/project.rs

```diff
@@ -237,6 +238,7 @@ impl DataStore {
         // Verify that child resources do not exist.
         self.ensure_no_instances_in_project(opctx, authz_project).await?;
         self.ensure_no_disks_in_project(opctx, authz_project).await?;
+        self.ensure_no_floating_ips_in_project(opctx, authz_project).await?;
```

Comment:
Sorry this wasn't flagged in the first PR, but hey, happy that this (at least *looks* like it) was easy to add!

Reply:
Thanks for the review; the existing machinery did indeed make it easy to add in!